### DISCUSSION TITLE
How do I Port Forward on my Ubuntu server?

### DISCUSSION THREAD

**Discussion by RavenX-13** [itzg/docker-minecraft-server#2185] [Q&A]
So i used the command given in the description and it works for private ip, but after adding rules to my firewall and router setting it still wont work. Im new into this and never used docker before. When i try to use <public ip>:25565 to connect to it it wont work

**Comment by itzg** [comment]
The remainder is not necessarily a docker specific problem; however, make sure you're not using docker provided by your Linux distribution but instead using Docker's instructions.

**Reply by RavenX-13** [reply]
I did install docker using docs.

**Reply by RavenX-13** [reply]
It does work when only using public ip and not the 25565 port but my pi hole which is on port 80 doesnt work anymore. What could be the problem? I want the server to use the 25565 port. To run it i used the "docker run -d -it -p 25565:25565 -e EULA=TRUE itzg/minecraft-server".

**Reply by itzg** [reply]
That is the correct setup for the container, so it's something non-Docker with your host/network. Sorry, I can't offer any further advice but maybe someone else can add ideas here.
